dvc/stage/params.py

```diff
@@ -12,11 +9,9 @@ class StageParams:
     PARAM_ALWAYS_CHANGED = "always_changed"
     PARAM_PARAMS = "params"
 
-
-class OutputParams(Enum):
-    PERSIST = "outs_persist"
-    PERSIST_NO_CACHE = "outs_persist_no_cache"
-    METRICS_NO_CACHE = "metrics_no_cache"
-    METRICS = "metrics"
-    NO_CACHE = "outs_no_cache"
-    OUTS = "outs"
+    PARAM_OUTS_PERSIST = "outs_persist"
```

Comment:
Btw, might be worth renaming it to `persist` instead of `outs_persist`. Looks a lot nicer that way.

Reply:
may be in this case we can make it a flag within the `outs` item?

Reply:
@shcheklein It is that already :) But we also keep `plain` format where it mimics CLI options. So I wonder if we should make it `persist` and `--persist` in `dvc run`.

Reply:
yep, sounds good to me! never liked these JAVA-style - `--outs-enterprisy-long-name-include-everything`  :)